src/components/CreateButton.tsx

```diff
@@ -375,7 +375,7 @@ const CreateButton = () => {
                     return new Promise<string>(async (resolve, reject) => {
                         const timeout = setTimeout(
                             () => reject(new Error(t("timeout"))),
-                            15_000,
+                            20_000,
```

Comment:
If it takes up to 18 seconds for you, let's add some more buffer and set it to 25 seconds
Also, let's put it in a constant

Reply:
Maybe we should bump LNURL fetch timeouts the same way. If a request resolves successfuly, that result will be used immediately. No harm in using the same high timeout for both